src/main/scala/org/broadinstitute/dsde/firecloud/webservice/RegisterApiService.scala

```diff
@@ -43,4 +43,14 @@ trait RegisterApiService extends Directives with RequestBuilding with StandardUs
         }
       }
     }
+
+  val tosRoutes: Route =
+    pathPrefix("tos") {
+      path("text") {
+        get {
+          complete {
+            registerServiceConstructor().getTermsOfServiceText }
```

Comment:
could this use the `passthrough` directive instead? Then you wouldn't need to add anything to `SamDAO`

Reply:
Oh cool! Testing this out-